lib/fathead/python_coverage/output.txt

```diff
@@ -0,0 +1,52 @@
+Coverage __init__	A										<section class="prog_container"><pre><code>__init__(data_file=None, data_suffix=None, cover_pylib=None, auto_data=False, timid=None, branch=None, config_file=True, source=None, omit=None, include=None, debug=None, concurrency=None)</pre></code><p>data_file is the base name of the data file to use, defaulting to\n”.coverage”.  data_suffix is appended (with a dot) to data_file to\ncreate the final file name.  If data_suffix is simply True, then a\nsuffix is created with the machine and process identity included.\ncover_pylib is a boolean determining whether Python code installed\nwith the Python interpreter is measured.  This includes the Python\nstandard library and any packages installed with the interpreter.\nIf auto_data is true, then any existing data file will be read when\ncoverage measurement starts, and data will be saved automatically when\nmeasurement stops.\nIf timid is true, then a slower and simpler trace function will be\nused.  This is important for some environments where manipulation of\ntracing functions breaks the faster trace function.\nIf branch is true, then branch coverage will be measured in addition\nto the usual statement coverage.\nconfig_file determines what configuration file to read:\n\n\nIf it is ”.coveragerc”, it is interpreted as if it were True,\nfor backward compatibility.\nIf it is a string, it is the name of the file to read.  If the\nfile can’t be read, it is an error.\nIf it is True, then a few standard files names are tried\n(”.coveragerc”, “setup.cfg”, “tox.ini”).  It is not an error for\nthese files to not be found.\nIf it is False, then no configuration file is read.\n\n\nsource is a list of file paths or package names.  Only code located\nin the trees indicated by the file paths or package names will be\nmeasured.\ninclude and omit are lists of file name patterns. Files that match\ninclude will be measured, files that match omit will not.  Each\nwill also accept a single string argument.\ndebug is a list of strings indicating what debugging information is\ndesired.\nconcurrency is a string indicating the concurrency library being used\nin the measured code.  Without this, coverage.py will get incorrect\nresults if these libraries are in use.  Valid strings are “greenlet”,\n“eventlet”, “gevent”, “multiprocessing”, or “thread” (the default).\nThis can also be a list of these strings.\n\nNew in version 4.0: The concurrency parameter.\n\n\nNew in version 4.2: The concurrency parameter can now be a list of strings.</p></section>	https://coverage.readthedocs.io/en/latest/api_coverage.html#coverage.Coverage.__init__
```

Comment:
Thanks @moollaza ,

Just to be clear, `<span>` blocks need to be to removed right?